web-ui/src/main/resources/catalog/views/default/config.js

```diff
@@ -95,7 +95,7 @@
             })
 
           };
-
+          
```

Comment:
Your IDE should be smart enough to remove trailing space. If not, you need to have a serious discussion with it. ;)